internal/pipe/snapcraft/snapcraft.go

```diff
@@ -231,11 +246,16 @@ func create(ctx *context.Context, snap config.Snapcraft, arch string, binaries [
 				WithField("name", name).
 				Debug("passed binary to snapcraft")
 
+			command := name
+			if config.Command != "" {
```

Comment:
Looks better, avoided it initially so as to not mutate the config

Reply:
So that change actually breaks the tests, and it feels like patchwork to reset the config in the tests for them to work properly.

Reverted, let me know if that is OK or how to proceed.

Reply:
https://github.com/goreleaser/goreleaser/runs/745188125?check_suite_focus=true